benchmark/startup.txt

```diff
@@ -1,15 +1,16 @@
-toywasm (default),0.17,0.16,0.00,40300544
```

Comment:
btw. if you rename this CVS file into `startup.cvs` (akin to `ffmpeg.cvs`) then GitHub can render it as such which is more readable. :)

Reply:
do you mean csv?

Reply:
yes sorry

Reply:
At least for me GitHub beautifully renders your `ffmpeg.csv`.
![image](https://github.com/yamt/toywasm/assets/8193155/bd328724-d9ec-4921-a7d3-b63c03d8202a)